gradoop-common/src/main/java/org/gradoop/common/model/impl/properties/PropertyValue.java

```diff
@@ -879,4 +892,15 @@ public String toString() {
       getObject().toString() :
       GConstants.NULL_STRING;
   }
+  
+  /**
+   * Throw a runtime exception if this property value can't be represented 
+   * in {@link PropertyValue#MAX_BINARY_LENGTH} bytes.
+   */
+  private void validateBytesLength() {
+    if(rawBytes != null && rawBytes.length> MAX_BINARY_LENGTH){
```

Comment:
add whitespace after `if` and before `>` and `{` not sure why checkstyle did not catch this oO